### PR TITLE
Fix logs exporter module name

### PR DESCRIPTION
**Pull request by elevenzhan**
Fix logs exporter module name and description

### CODE REVIEW

**Review by jack-berg — APPROVED** [review]
Thanks!

**Review by breedx-splk — APPROVED** [review]
Yikes. 😁 Thanks thanks.

File: exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import io.opentelemetry.exporter.otlp.internal.logs.LogsRequestMarshaler;
 import java.io.InputStream;
 
-// Adapted from the protoc generated code for LogsServiceGrpc.
+/** Adapted from the protoc generated code for LogsServiceGrpc. */
```
Comment (anuraaga) [inline exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java:20]: This isn't really javadoc so a comment it more appropriate. Let's revert this change

Comment (jkwatson) [reply exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java:20]: I realized my comment was in "pending status". 🤦🏽

Comment (elevenzhan) [reply exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java:20]: Thanks, but I think this style is aligned with other source codes in project; 
In addition, Javadoc single-line comment can not pass spotlessJavaCheck, just like this:
```
/**
 * Adapted from the protoc generated code for LogsServiceGrpc.
 */
```
Do I need revert this change or keep it, @anuraaga @jkwatson

Comment (anuraaga) [reply exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java:20]: > In addition, Javadoc single-line comment can not pass spotlessJavaCheck, just like this:

I don't follow this - spotless is currently passing right? Let's go ahead and rever tthis - this is an internal class so it doesn't need real javadoc, this is an implementation detail comment targeted at code authors, not docs for end users. We like to make that distinction where possible.

Comment (elevenzhan) [reply exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/MarshalerLogsServiceGrpc.java]: I see, already revert

**Review by jkwatson — APPROVED** [review]
